console/console-init/ui/src/modules/msg-and-iot/dailogs/CreateProjectContainer/CreateProjectContainer.tsx

```diff
@@ -60,7 +60,7 @@ const CreateProjectContainer: React.FunctionComponent = () => {
     setIsCreatedSuccessfully(true);
   };
 
-  const [setQueryVariables] = useMutationQuery(
+  const [setMessagingVariables] = useMutationQuery(
```

Comment:
what you think about query name like   [setMessagingQueryVariables]

Reply:
I will rename it in future, once we start integrating with mock server